sdk/storage/storage-file-datalake/CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Release History
 
+## 12.4.0-beta.1 (2021-02-09)
+
+- Added support for Container Soft Delete. You can restore a deleted filesystem via `DataLakeServiceClient.undeleteFileSystem()`. And the `DataLakeServiceClient.listFileSystems()` now support an `includeDeleted` option to include soft deleted filesystems in the response.
+- Added `fromConnectionString` to `DataLakeServiceClient` to support construction from a connection string. Fixed bug [13396](https://github.com/Azure/azure-sdk-for-js/issues/13396).
```

Comment:
Why this line is moved from 12.3.1 to 12.4.0-beta.1?

Reply:
It was auto-merged and added to the wrong place.